### PR TITLE
PPTP-1872 - Link content and behaviour updates

### PR DESCRIPTION
**Pull request by PSouthern**
### Description of Work carried through

**/manufactured-components**

Link Behaviour: open in existing tab
Link Content: See [examples of components].

**/manufactured-weight**

Link Behaviour: open in existing tab
Link Content: See [examples of things you should not include].

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

**Review by tonymccrae-ee — APPROVED** [review]
Content only. Placeholders seem to be balanced.
All good.

File: conf/messages.en [hunk 1]
```diff
@@ -126,8 +127,8 @@ returns.manufacturedPlasticWeight.hint = Do not include plastic packaging compon
 returns.manufacturedPlasticWeight.hint.1 = used in the long-term storage of goods
 returns.manufacturedPlasticWeight.hint.2 = an integral part of the goods
 returns.manufacturedPlasticWeight.hint.3 = reused for the presentation of goods
-returns.manufacturedPlasticWeight.hint.4 = See
-returns.manufacturedPlasticWeight.hint.4.link = examples of things you should not include.
+returns.manufacturedPlasticWeight.hint.4 = See {0} .
```
Comment (RyanKellyHMRC) [inline conf/messages.en:130]: Do we really want the space before the fullstop?

**Review by RyanKellyHMRC — APPROVED** [review]
Looks good. Just querying the space before the fullstop.

**Review by RyanKellyHMRC — APPROVED** [review]
Nice one!

**Pull request merged**